### ISSUE TITLE
번역 질문

### ISSUE THREAD

**Issue by fuzzythecat** [KerasKorea/KEKOxTutorial#167]
번역중에 질문이 생겨 이슈 올립니다.
Polynomial learning rate decay는 어떻게 번역하는게 맞을까요? 🤕

**Comment by mike2ox** [comment]
제가 예전에 `exponential learning rate decay` 를 `학습율의 기하급수적 감소` 로 이해했던걸 적용하면
`학습율의 다항적 감소`가 어떨까요?

**Comment by fuzzythecat** [comment]
감사합니다 `학습률의 다항적 감소(Polynomial Learning Rate Decay)`로 번역하였습니다 👍
